src/MeshKernel/CurvilinearGrid/CurvilinearGridFromPolygon.cpp

```diff
@@ -55,8 +55,8 @@ CurvilinearGrid CurvilinearGridFromPolygon::Compute(size_t firstNode,
     }
 
     // for the current polygon find the number of nodes
-    auto start = m_polygon->m_outer_polygons_indices[0].first;
-    auto end = m_polygon->m_outer_polygons_indices[0].second;
+    auto start = m_polygon->OuterIndices(0).first;
```

Comment:
can use here structured bindings?

Reply:
Of course, I think we there are other places in this commit where it can be also used. I'll double check.

Reply:
Turns out it's the only instance in this commit.